docs/en/latest/plugins/ip-restriction.md

```diff
@@ -70,6 +70,20 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
+If you want to use a custom message, you can configure it in the plugin section.
+
+```shell
+"plugins": {
+    "ip-restriction": {
+        "whitelist": [
+            "127.0.0.1",
+            "113.74.26.106/24"
+        ],
+        "message": "Do you want to do something bad?"
```

Comment:
Need to update `Attributes`

Reply:
Sorry, I didn't understand😥, what does this attribute refer to?

Reply:
https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/ip-restriction.md#attributes
Need to update the table.

Reply:
Thanks, now I understand😊,previously ignored this place